pivorakmeetup/pivorak-web-app
# Bump ruby version to 2.6.5

### Description

Update ruby version to 2.6.5

### How to test
- Visit https://pivorak-staging.herokuapp.com/
- Verify it is working
